packages/desktop-gui/src/lib/ipc-bus.js

```diff
@@ -108,6 +96,16 @@ const ipcBus = (...args) => {
     }
   }
 
+  // remove nodes that cannot be cloned
+  // https://github.com/cypress-io/cypress/issues/6750
+  args = args.map((arg) => {
```

Comment:
can we move the callback into a named function please?

Reply:
and I think the map could avoid extra function then
```
const nullInvalidValue = (val) => {
  if (_.isFunction(val) || _.isElement(val)) {
    return null
  }
}
const cleanArgument = _.partialRight(_.cloneDeepWith, nullInvalidValue)
args = args.map(cleanArgument)
```

Reply:
extracted to named function, also switched references to `remove` to `nullify`, since it doesn't technically remove unserializable values, it makes them `null`